reg_tests/executeOpenfastRegressionCase.py

```diff
@@ -24,9 +24,10 @@
 
 import os
 import sys
-basepath = os.path.dirname(os.path.abspath(__file__))
+basepath = os.path.dirname(__file__)
```

Comment:
Is there a reason for reverting this change from https://github.com/OpenFAST/openfast/pull/1021?

Reply:
Strange that all those got changed back. Looking into what happened.